src/Proto.Remote/EndpointWatcher.cs

```diff
@@ -56,20 +56,17 @@ public Task ReceiveAsync(IContext context)
                     _watched[msg.Watcher.Id] = null;
 
                     var w = new Unwatch(msg.Watcher);
-                    RemoteProcess.SendRemoteMessage(msg.Watchee, w);
+                    RemoteProcess.SendRemoteMessage(msg.Watchee, w,Serialization.DefaultSerializerId);

```

Comment:
When sending remote messages explicitly w/o using a PID, we need to state what serializer to use